### ISSUE TITLE
Ask for tips in conformer model training

### ISSUE THREAD

**Issue by mr-coconut** [huseinzol05/malaya-speech#24]
Hi~~~
Do you perhaps have any tips in model training on the telephony speech datasets? Like, Do we need noise reduction/speech enhancement on telephony data before inputting data into the training process? How long should the training data be in general? How many epochs/rounds are typically needed? 
Thanks~~~

**Comment by huseinzol05** [comment]
u should do noise augmentation to get wider audio definition, eg,

1. if original sr is 16k, reduce it to 4k, upsampling back to 16k
2. room echo
3. loudness

or can try, https://iver56.github.io/audiomentations/

I trained those models until test loss hit plateau.
